### PR TITLE
chore: align to celestia node v0.11-rc

### PR DESCRIPTION
**Pull request by zvolin**

### CODE REVIEW

File: ci/Dockerfile.bridge [hunk 1]
```diff
@@ -8,8 +8,9 @@ ENV CELESTIA_HOME=/root
 RUN apk update && apk add --no-cache bash jq
 
 # Copy in the binary
-COPY --from=ghcr.io/celestiaorg/celestia-node:4b96022 /bin/celestia /bin/celestia
-COPY --from=ghcr.io/celestiaorg/celestia-node:4b96022 /bin/cel-key /bin/cel-key
+# v0.11.0-rc13
+COPY --from=ghcr.io/celestiaorg/celestia-node:8ca777e /bin/celestia /bin/celestia
+COPY --from=ghcr.io/celestiaorg/celestia-node:8ca777e /bin/cel-key /bin/cel-key
```
Comment (oblique) [inline ci/Dockerfile.bridge:13]: we can actually use the tag

```suggestion
COPY --from=ghcr.io/celestiaorg/celestia-node:v0.11.0-rc13 /bin/celestia /bin/celestia
COPY --from=ghcr.io/celestiaorg/celestia-node:v0.11.0-rc13 /bin/cel-key /bin/cel-key
```

File: ci/Dockerfile.validator [hunk 2]
```diff
@@ -8,7 +8,8 @@ ENV CELESTIA_HOME=/root
 RUN apk update && apk add --no-cache bash jq
 
 # Copy in the binary
-COPY --from=ghcr.io/celestiaorg/celestia-app:02f37a3 /bin/celestia-appd /bin/celestia-appd
+# v1.0.0-rc14
+COPY --from=ghcr.io/celestiaorg/celestia-app:cdc6bff /bin/celestia-appd /bin/celestia-appd
```
Comment (oblique) [inline ci/Dockerfile.validator:12]:
```suggestion
COPY --from=ghcr.io/celestiaorg/celestia-app:v1.0.0-rc14 /bin/celestia-appd /bin/celestia-appd
```

File: types/src/blob.rs [hunk 3]
```diff
@@ -9,6 +9,15 @@ use crate::consts::appconsts;
 use crate::nmt::Namespace;
 use crate::{bail_validation, Error, Result};
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct SubmitOptions {
+    // TODO: giving negative fees always results in insufficient fee err, but celestia
+    //       have it like so, should we do a u64 there?
```
Comment (fl0rek) [inline types/src/blob.rs:16]: I see `int64` there, is celestia able to correctly reject `i64::MAX..u64::MAX` values? If so I don't think there's harm to it

File: ci/Dockerfile.bridge (hunk 1 above)
Comment (zvolin) [reply ci/Dockerfile.bridge:13]: I don't think they tag images but will check

Comment (zvolin) [reply ci/Dockerfile.bridge:13]: indeed, thanks

File: types/src/blob.rs (hunk 3 above)
Comment (oblique) [reply types/src/blob.rs:16]: Actually in other places we used `Uint` for fees and it was serialized as numeric strings. What celestia does here? I prefer `u64` if we can not use `Uint` and have a `serialize_with` that checks if the number is bigger than `i64`. Or maybe open a bug to them.

Comment (oblique) [reply types/src/blob.rs:16]: actually having `u64` at let the server return an error is more acceptable for me than `i64`.

Comment (zvolin) [reply types/src/blob.rs:16]: unfortunately requests with value in this range just fails, I'll stick to the exact types celestia is using and delete this todo

**Review by fl0rek — APPROVED** [review]
:guitar:

**Review by oblique — APPROVED** [review]

**Pull request merged**